### DISCUSSION TITLE
Navigation Parent Item - Icon MixMux

### DISCUSSION THREAD

**Discussion by hofstein** [filamentphp/filament#11297] [General]
Hey there,

I want to build a Navigation with 3 levels.

Group
-- Navigation Item
---- Child Item

I want to have one place where I give the group an icon. But I think it is not possible, or?

1. I give only the group an icon .. i get this error:
Navigation item [...] has child items but no icon. Parent items must have an icon to ensure a proper user experience

2. I give the group and the "Navigation Item" an icon .. i get this error:
Navigation group [...] has an icon but one or more of its items also have icons. Either the group or its items can have icons, but not both. This is to ensure a proper user experience.

The only way to make it work it is to give the icon only to "Navigation Item". But in my case every Navigation Item is a composer package and I don't know which ones are active. So I have to give every Navigation Item the same icon to have an icon for the group.

Is there a better solution?

Regards
Marcus

**Comment by saade** [comment]
https://filamentphp.com/docs/3.x/panels/navigation#customizing-navigation-groups

**Reply by renanmachad** [reply]
This dosnt give any awnser

**Comment by appstarbd** [comment]
Follow: https://filamentphp.com/docs/3.x/panels/resources/getting-started#grouping-resource-navigation-items-under-other-items
Also add same navigation group name "protected static ?string $navigationGroup = 'Group Name';" in both resources.
